py/tests/integration/setup.yaml

```diff
@@ -41,16 +41,14 @@ tests: # test specific settings
       nodes:
         bob: dev1
         alice: dev2
-      blocks_to_post: 7
-      chain_data: py/tests/integration/data/bchain.txt
```

Comment:
@velzevur I am double checking the deletion of the pre-mined chain files.

Reply:
It looks still there.
```
$ find . -name bchain.txt
./py/tests/integration/data/bchain.txt
$ ls -lh ./py/tests/integration/data/bchain.txt
-rw-r--r--  1 luca  staff   7.9K 16 Nov 09:34 ./py/tests/integration/data/bchain.txt
```

I see it is used by [`test_ext_api`](https://github.com/aeternity/epoch/pull/375/files#diff-76bf7aa6bd9ae9d90af0e0d0f5156921R36).

Reply:
yes, for this we will need some premined chain; this functionality is covered in the user acceptance tests; I'll fix the API calls in a subsequent PR